sdk/search/Azure.Search.Documents/README.md

```diff
@@ -171,19 +174,17 @@ For more details about Dependency Injection in ASP.NET Core apps, see [Dependenc
 
 ## Key concepts
 
-An Azure AI Search service contains one or more indexes that provide
+An Azure Cognitive Search service contains one or more indexes that provide
```

Comment:
Should we not take these changes as they are part of the rebranding?

Reply:
Updated!